src/main/scala/nl.knaw.dans.easy.solr4files/EasySolr4filesIndexApp.scala

```diff
@@ -35,7 +35,9 @@ trait EasySolr4filesIndexApp extends ApplicationWiring with AutoCloseable
   }
 
   def initSingleStore(storeName: String): Try[StoreSubmitted] = {
-    getBagIds(storeName)
+    val connTimeOutMs = configuration.properties.getInt("list-bags.connection-timeout-ms")
```

Comment:
If `list-bags.connection-timeout-ms` is misconfigured to contain a non-integer value the resulting exception will not be caught.

Reply:
@janvanmansum you are right, especially since this will only break the application once the `initSingleStore` call is done. Below I suggested a different approach that is (1) more in line with the rest of this application and (2) fixes the problem you mentioned here.